### PR TITLE
chore: add package publish nightly

### PR DESCRIPTION
**Pull request by aseaday**

### CODE REVIEW

**Comment by aseaday** [comment]
Screen Shot 2022-10-15 at 01 58 39

File: src/cli/main.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use clap::{value_parser, App, Arg, SubCommand};
+use clap::{value_parser, App,AppSettings, Arg, SubCommand};
```
Comment (lanbones) [inline src/cli/main.rs:1]: please fmt

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: release
+on:
+  schedule:
+    - cron: '0 2 * * *' # UTC 2:00 every day
```
Comment (lanbones) [inline .github/workflows/release.yml:4]: @xgaozoyoe Do we plan to publish package now? I think our code is still in an early stage.

File: src/cli/main.rs (hunk 1 above)
Comment (aseaday) [reply src/cli/main.rs]: Fixed and thx!

File: .github/workflows/release.yml (hunk 2 above)
Comment (aseaday) [reply .github/workflows/release.yml:4]: It is only the cli tools for users to easily test.

Comment (xgaozoyoe) [reply .github/workflows/release.yml:4]: let's wait for another 2 months before publishing I guess. We can mark this PR WIP.

Comment (lanbones) [reply .github/workflows/release.yml:4]: I prefer to keep the workflow_dispatch and remove the schedule.

Comment (aseaday) [reply .github/workflows/release.yml:4]: > I prefer to keep the workflow_dispatch and remove the schedule.

LGTM, WDYT @xgaozoyoe

Comment (xgaozoyoe) [reply .github/workflows/release.yml:4]: shall we remove the schedule but manually do a release instead?

**Review by lanbones — APPROVED** [review]

**Pull request merged**